### PR TITLE
update the renewals page to work with the current API version

### PR DESCRIPTION
**Pull request by Schparky**
### Removed
- Removed the ledger list from the annual renewals page.
- Removed the download file from the annual renewals page.
- Removed `LedgerList` component.
- Removed `getPolicyRenewals` function.

### Added
- Added a Process button to the annual renewals page.

### CODE REVIEW

File: src/pages/admin/renewals.svelte [hunk 1]
```diff
@@ -1,51 +1,25 @@
...
+metatags.title = formatPageTitle('Admin > Renewals')
 </script>
 
 <Page>
-  <div class="flex justify-between align-items-center">
-    <Button raised on:click={onUpdate}>Update</Button>
-    <Button disabled={!reportFile?.url} on:click={onDownload}>Download</Button>
+  <h3>Annual renewals</h3>
+  <p>annual process for renewing coverage</p>
+  <p></p>
```
Comment (hobbitronics) [inline src/pages/admin/renewals.svelte:21]: what is this empty p tag for?

Comment (Schparky) [reply src/pages/admin/renewals.svelte:21]: It can be removed. I'll leave it though, since the next PR resolves this.

Comment (hobbitronics) [reply src/pages/admin/renewals.svelte:21]: I see.

**Review by hobbitronics — APPROVED** [review]
Looks good.

**Pull request merged**